### ISSUE TITLE
[BUG] (Account is still loading) message always when I switch from my LND-LNC account to Alby account

### ISSUE THREAD

**Issue by bitcoinuser** [getAlby/lightning-browser-extension#2275]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Hi,

(Account is still loading) message always when I switch from my LND-LNC account to Alby account.

https://user-images.githubusercontent.com/1178856/227812474-cc13872f-cdf9-4884-9127-9b19168e68d5.mp4

### Steps To Reproduce

1 - You need to have Umbrel LND-LNC account and Alby custodial account
2- Switch from Umbrel LND-LNC account to Alby Account.

### Expected behavior

No Account is still loading message.

### Alby information

Alby: 1.28.1

### Are you working on this?

No

**Comment by Ladyperpetua** [comment]
hi can i please be assigned this issue?

**Comment by rolznz** [comment]
Hi @Ladyperpetua 

Thanks for being willing to work on this issue.

Are you able to reproduce it locally following bitcoinuser's instructions?

**Comment by Ladyperpetua** [comment]
i'm still on it but i havent reproduced it yet?

**Comment by rolznz** [comment]
@Ladyperpetua was the connection working previously? where is your LND node connected to? https://lnd1.regtest.getalby.com? 

https://github.com/getAlby/lightning-browser-extension/wiki/Test-setup

**Comment by rolznz** [comment]
@Ladyperpetua oh, I see. You will probably need to run your own node to reproduce this. I think you should be able to reproduce it before working on the task.

**Comment by Ladyperpetua** [comment]
Yes, my connection was fine, I tried other alternative using a different browser, and i created an eclair node through polar.  THE LND node just wasn't working

**Comment by Ladyperpetua** [comment]
I also wanted to ask if i could work on it without getting assigned?

**Comment by rolznz** [comment]
@Ladyperpetua sure, that's fine.
